roles/ceilometer-common/templates/etc/ceilometer/event_definitions.yaml

```diff
@@ -0,0 +1,62 @@
+---
```

Comment:
I believe the reasoning for having it here was to consolidate all ceilometer input templates in one place, which then makes it easy to copy over to the nodes in a single task